taxonomies/additives.txt

```diff
@@ -2688,7 +2687,8 @@ ro:E162, Roșu de sfeclă, betanină
 sh:E162, Betanin
 sk:E162, Cviklová červená, betanín
 sl:E162, Betalain, betanin
-sv:E162, Rödbetsrött, betanin, E 162
+sv:rödbetsrött
```

Comment:
```suggestion
sv:rödbetsrött, betanin, E 162
```

Reply:
for additives, the e number needs to be the first name in all languages.